samples/servicecontrol/adapter-sqlserver/sample.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Using adapter to centralize monitoring of multi-instance SQL Server
+component: SCTransportAdapter
+reviewed: 2017-05-11
+related:
+ - servicecontrol
+ - servicecontrol/plugins
+---
+
+
+The purpose of the adapter is to isolate ServiceControl from specifics of physical deployment topology of the business endpoints (such as [multi-instance](/nservicebus/sqlserver/deployment-options.md#modes-overview-multi-instance) mode. In order to do so, the adapter provides a ServiceControl interface to the business endpoints.
+
+
+## Prerequisistes
+
+ 1. [Install ServiceControl](/servicecontrol/installation.md).
+ 1. Create `ServiceControl` database on the local SQL Server instance. 
+ 1. Using [ServiceControl Management](/servicecontrol/license.md#servicecontrol-management-app) tool, set up ServiceControl to monitor endpoints using SQL Server transport:
+	 
+	* Add a new ServiceControl instance: 
+	   * Use `Particular.ServiceControl.SQL` as the instance name (make sure there is no other instance of SC running with the same name).
+	   * Use "User" account and provide credentials to allow for integrated authentication.
+	   * Specify "Data Source=.\SQLEXPRESS;Initial Catalog=ServiceControl;Integrated Security=True;Max Pool Size=100;Min Pool Size=10" as a connection string. ServiceControl Manager will automatically create queue tables in the database.
```

Comment:
Note: If other ServiceControl instances have been running on this machine, it's necessary to specify a non-default port number for API. [Adjust ServicePulse settings](/servicepulse/host-config.md#changing-the-servicecontrol-url) accordingly to point to this location.